io_scene_godot/structures.py

```diff
@@ -59,12 +61,14 @@ def get_internal_resource(self, hashable):
 
     def add_internal_resource(self, item, hashable):
         """See comment on external resources. It's the same"""
-        if self.get_internal_resource(hashable) is not None:
+        if (hashable is not None and
```

Comment:
So I guess you wanted to be able to pass in hashable=None to be able to force it to create a new resource. What is the use case of this? The idea is that every time you can pass in the blender _something_ that is being stored.

Reply:
You can pass in a reference to a blender object, so you could pass in a bpy.types.Action as the hashable (yes, it's mutable, but it's a C-based structure, so python treats it's memory address as the hash)

Reply:
In discussion via IRC this was resolved in favor of the existing code because while an f-curve or action may be unique, there is nothing that represents Godot's AnimationPlayer node.

Reply:
If you agree I still want to provide no hash for an animation resource. Use a bpy.types.action as a hash object is hacky (it does not one-one mapping to the animation resource) and also no good (animation is exporting in single traversal of blender tree, it never processes an action more that one time, so use a action would not catch any duplication)